pkg/query/compression/compression.go

```diff
@@ -0,0 +1,79 @@
+package compression
```

Comment:
Do we need this? IIRC the standard library negotiates and implements gzipping by default nowadays.

Reply:
yes, this might be redundant, will kill it